src/main/sensors/acceleration_init.c

```diff
@@ -35,21 +35,64 @@
 #include "config/feature.h"
 
 #include "drivers/accgyro/accgyro.h"
-#include "drivers/accgyro/accgyro_virtual.h"
 #include "drivers/accgyro/accgyro_mpu.h"
-#include "drivers/accgyro/accgyro_mpu3050.h"
-#include "drivers/accgyro/accgyro_mpu6050.h"
-#include "drivers/accgyro/accgyro_mpu6500.h"
+
+#ifdef USE_ACCGYRO_BMI160
```

Comment:
I'm not sure about these `#if`s .. .They make some sense here (single file where virtual binding is defined). but they are ugly and a bit error prone.
Including .h files unconditionally will only make compile-time error link-time one. And it will remove some duplication.

So this PR *may* go other way around and remove `#ifdef USE_ACC_ADXL345` etc ..

Eventually, it is possible to refactor sensor initialization into `.section` that will collect all enabled gyro/acc/mag/... drivers and make initialization agnostic to actual sensors compiled in/ enabled. (It does work quite fine for for PGs). Something like 
```
struct {
  uint8_t sensor_type;    // acc, gyto, baro, ... 
  uint8_t sensor_driver;  // driver class where common driver is used for multiple devices
  uint8_t sensor_id;        // accelerationSensor_e / gyroHardware_e / ...
  
  sensor_init_t *init;
} sensor_registration_t;
```

It will also simplify MSP/CLI introspection (list available sensors) etc ...

Reply:
Will leave the .section for another PR.

Reply:
It wasn't meant to be implemented instead of this PR ... just an idea about possible way forward; it will make code improved with this PR obsolete ..

Reply:
So revert last commit ?